Makefile

```diff
@@ -11,10 +12,16 @@ verify:
 build:
 	docker build -t "$(IMAGE_NAME):$(IMAGE_TAG)" .
 
+tag:
+	docker tag "$(IMAGE_NAME):$(IMAGE_TAG)" "$(REPO_NAME)/$(IMAGE_NAME):$(IMAGE_TAG)"
+
+push:
+	docker push "$(REPO_NAME)/$(IMAGE_NAME):$(IMAGE_TAG)"
+
```

Comment:
note to self to have some developer docs (we need to keep this image up to date and in the right place)